fdbserver/QuietDatabase.actor.cpp

```diff
@@ -673,8 +673,6 @@ ACTOR Future<Void> repairDeadDatacenter(Database cx,
 			    .detail("RemoteDead", remoteDead)
 			    .detail("PrimaryDead", primaryDead);
 			g_simulator->usableRegions = 1;
-			g_simulator->killDataCenter(
-			    primaryDead ? g_simulator->primaryDcId : g_simulator->remoteDcId, ISimulator::KillInstantly, true);
```

Comment:
The original description of this statement:
```
When primary and remote regions use the same dcid,
the canKillProcesses does not give the right protection.
```
By reading the source of `killDataCenter` in `sim2.actor.cpp`, since we set `forceKill` to be `true`, there is no function call to `canKillProcesses`. I do not think the description makes sense. Also I do not understand the relationship between the description and the code change.